composer.json

```diff
@@ -29,11 +29,11 @@
     "require-dev": {
         "roave/security-advisories": "dev-latest",
         "codeception/codeception": "4.1.*",
```

Comment:
How about supporting codeception/codeception 4.2?

Reply:
Hi @Naktibalda you're right! I will change it.